packages/web-react/src/components/FileUploader/demo/FileUploaderInputMultipleWithFileQueueBehaviorControl.tsx

```diff
@@ -15,37 +11,37 @@ const Story = (args: FileUploaderBaseProps) => {
 
   return (
     <FileUploader
-      {...args}
-      id="fileUploaderExampleDismissible"
-      onDismiss={onDismiss}
-      fileQueue={fileQueue}
       addToQueue={addToQueue}
       clearQueue={clearQueue}
+      fileQueue={fileQueue}
       findInQueue={findInQueue}
+      id="fileUploaderMultipleWithQueueBehaviorControl"
+      onDismiss={onDismiss}
       updateQueue={updateQueue}
     >
       <FileUploaderInput
-        id="fileUploaderExampleDismissibleInput"
-        name="attachments"
+        accept=".png,image/jpeg"
+        helperText="Max size of each file is 10 MB"
+        id="fileUploaderMultipleWithQueueBehaviorControlInput"
+        isMultiple
         label="Label"
-        linkText="Upload your file(s)"
         labelText="or drag and drop here"
-        helperText="Max file size is 10 MB"
-        validationText="Validation message"
-        maxUploadedFiles={3}
+        linkText="Upload your file(s)"
+        maxUploadedFiles={2}
+        name="attachments"
         /* eslint-disable-next-line no-console */
         onError={(error) => console.error('My error log', error)}
         queueLimitBehavior="hide"
-        isMultiple
+        validationText="Validation message"
```

Comment:
by the way does it make sense to keep validationText here if it doesn't work without validationstate? Especially in only visual demos.

Reply:
probably not, i will remove it